README.md

```diff
@@ -109,41 +111,62 @@ app.run()                     # opens a window and starts the game.
 
    To close the window, you can by default, press shift+q or press the red x. to disable this, write `window.exit_button.enabled = False` somewhere in your code.
 
+## Documentation
+If you need help, you can watch documentation sites:

```

Comment:
Can you please fix the grammar here?

Reply:
Sorry, I'm not that good in english, what is the error here?

Reply:
I tried to change it to "If you need help, you can refer to the following documentation sites:"